### ISSUE TITLE
API down

### ISSUE THREAD

**Issue by samized** [seboschtion/NzzApp#4]
Wie es scheint wurde die API von NZZ abgeschaltet. Das ist wohl das definitive Ende dieser App... schade...

**Comment by seboschtion** [comment]
Ja, schade. Danke für die Information, ich habe die App vom Store genommen.
